docs/logging.md

```diff
@@ -0,0 +1,151 @@
+
+
+The following section will describe how to set up a Serverless (Knative) Cluster using `tmux`. This section follows the steps from the Quickstart guide with minor modifications for the log generation and running commands without `screen`. We present how to set up a multi-node cluster, however, the same modifications can be used to generate and extract logs from a single-node cluster.
+
+### 1. Setup All Nodes
+**On each node (both master and workers)**, execute the following instructions **as a non-root user with sudo rights** using **bash**:
+1. Open a new tmux session
+   ```bash
+    tmux new -s vhiveSession
+    ```
+3. Clone the vHive repository
+    ```bash
+    git clone --depth=1 https://github.com/vhive-serverless/vhive.git
+    ```
+2. Change your working directory to the root of the repository:
+    ```bash
+    cd vhive
+    ```
+3. Create a directory for vHive logs:
+    ```bash
+    mkdir -p /tmp/vhive-logs
+    ```
+3. Run the node setup script:
+    ```bash
+    ./scripts/cloudlab/setup_node.sh > >(tee -a /tmp/vhive-logs/setup_node.stdout) 2> >(tee -a /tmp/vhive-logs/setup_node.stderr >&2)
+    ```
+    > **BEWARE:**
+    >
+    > This script can print `Command failed` when creating the devmapper at the end. This can be safely ignored.
+
+### 2. Setup Worker Nodes
+**On each worker node**, execute the following instructions **as a non-root user with sudo rights** using **bash**:
+1. Run the script that setups kubelet:
+    ```bash
+    ./scripts/cluster/setup_worker_kubelet.sh > >(tee -a /tmp/vhive-logs/setup_worker_kubelet.stdout) 2> >(tee -a /tmp/vhive-logs/setup_worker_kubelet.stderr >&2)
+    ```
+2. Start `containerd`:
+    ```bash
+    sudo containerd 2>&1 | tee /tmp/vhive-logs/containerd_log.txt
+    ```
+    
+    > **Note:**
+    >
+    > The logs from `containerd` will be generated at this path `/tmp/vhive-logs/containerd_log.txt`. You can change the log's file name or directory. The same applies for all the log files used in the following commands.
+    
+3. Open a new tmux pane with <kbd>Ctrl</kbd>+<kbd>B</kbd> then <kbd>"</kbd> or <kbd>Ctrl</kbd>+<kbd>B</kbd> then <kbd>%</kbd> then start `firecracker-containerd`:
+     > **Note:**
+    > You use <kbd>Ctrl</kbd>+<kbd>B</kbd> then <kbd>"</kbd> or <kbd>Ctrl</kbd>+<kbd>B</kbd> then <kbd>%</kbd> to split the current `tmux` pane horizontally or vertically. Additionally you can use <kbd>Ctrl</kbd>+<kbd>B</kbd> then <kbd>→</kbd>, <kbd>←</kbd>, <kbd>↑</kbd> or <kbd>↓</kbd> to navigate between panes.
+
+    ```bash
+    sudo PATH=$PATH /usr/local/bin/firecracker-containerd --config /etc/firecracker-containerd/config.toml 2>&1 | tee /tmp/vhive-logs/firecracker-containerd_log.txt
+    ```
```

Comment:
This can be done without user's interaction in the command line. Here is an example command.
```
tmux new -s firecracker -d
tmux send -t firecracker "sudo PATH=$PATH /usr/local/bin/firecracker-containerd --config /etc/firecracker-containerd/config.toml 2>&1 | tee ~/firecracker_log.txt" ENTER
```